README.md

```diff
@@ -91,7 +91,7 @@ dsm_agent_download_port | Integer | The port to connect to the Deep Security Man
 force_reactivation | Boolean | Force re-activation even if the Deep Security Agent has already been activated. | false
 group_id | String | The Deep Security ID assigned to the computer group and applied to the agents on activation. | 1
 policy_id | String | ID of the Deep Security Policy to be assigned to the agents. | 1
-policy_name | String | Name of the Deep Security Policy to be assigned to the agents. | Base Policy
+policy_name | String | Name of the Deep Security Policy to be assigned to the agents. Policy_name Will be ignored if policy_id has being set. | Base Policy
```

Comment:
"Policy_name" need to be lowercase, and "Will" should be lowercase as well.
Maybe it would be better to mark the note as **Will be ignored if policy_id has being set.**